cmd/gossamer/configcmd_test.go

```diff
@@ -383,16 +357,12 @@ func TestSetRpcConfig(t *testing.T) {
 		c := c // bypass scopelint false positive
 		t.Run(c.description, func(t *testing.T) {
 			context, err := createCliContext(c.description, c.flags, c.values)
-			if err != nil {
-				t.Fatal(err)
-			}
+			require.Nil(t, err)
 
 			input := cfg.DefaultConfig()
 			setRpcConfig(context, &input.Rpc)
 
-			if !reflect.DeepEqual(input.Rpc, c.expected) {
-				t.Fatalf("\ngot %+v\nexpected %+v", input.Rpc, c.expected)
-			}
+			require.Equal(t, c.expected, input.Rpc)
```

Comment:
This is nice. Does it accomplish the same level of depth as `reflect.DeepEqual`?

Reply:
yep